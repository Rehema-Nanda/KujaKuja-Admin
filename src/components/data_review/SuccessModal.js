import React from "react";
import { Modal, ModalBody, ModalHeader } from "reactstrap";

const checkIcon = require("../../img/icons/check_mark.svg");

const SuccessModal = ({
    isOpen, fixedDataName, numberOfFixedResponses, toggle,
}) => {
    return (
        <Modal
            isOpen={isOpen}
            size="lg"
            className="daily-review-page"
            style={{ maxWidth: "68.5rem" }}
        >
            <ModalHeader toggle={toggle}>Success!</ModalHeader>
            <ModalBody>
                <div>
                    <img alt="" src={checkIcon} className="success-modal-check-mark" />
                    <p className="success-modal-check-text">
                        {fixedDataName}
                        {" "}
                        has been fixed on
                        {" "}
                        {numberOfFixedResponses}
                        {" "}
                        response(s)
                    </p>
                </div>
            </ModalBody>
        </Modal>
    );
};

export default SuccessModal;
