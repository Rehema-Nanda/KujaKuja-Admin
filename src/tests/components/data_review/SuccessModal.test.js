import React from "react";
import { shallow } from "enzyme";
import SuccessModal from "../../../components/data_review/SuccessModal";

describe("success modal", () => {
    it("renders correctly", () => {
        const tesProps = {
            toggle: jest.fn(),
            isOpen: true,
            fixedDataName: "Service Point",
            numberOfFixedResponses: 3,
        };
        const wrapper = shallow(<SuccessModal {...tesProps} />);

        expect(wrapper.debug()).toMatchSnapshot();
    });
});
