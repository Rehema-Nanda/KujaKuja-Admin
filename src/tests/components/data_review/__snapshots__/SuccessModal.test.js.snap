// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`success modal renders correctly 1`] = `
"<Modal isOpen={true} size=\\"lg\\" className=\\"daily-review-page\\" style={{...}} autoFocus={true} centered={false} role=\\"dialog\\" backdrop={true} keyboard={true} zIndex={1050} fade={true} onOpened={[Function: noop]} onClosed={[Function: noop]} modalTransition={{...}} backdropTransition={{...}}>
  <ModalHeader toggle={[Function: mockConstructor]} tag=\\"h5\\" wrapTag=\\"div\\" closeAriaLabel=\\"Close\\" charCode={215}>
    Success!
  </ModalHeader>
  <ModalBody tag=\\"div\\">
    <div>
      <img alt=\\"\\" src={{...}} className=\\"success-modal-check-mark\\" />
      <p className=\\"success-modal-check-text\\">
        Service Point
         
        has been fixed on
         
        3
         
        response(s)
      </p>
    </div>
  </ModalBody>
</Modal>"
`;
